Dfe.Academies.Academisation.WebApi/Program.cs

```diff
@@ -82,6 +82,10 @@
 		Title = "Academisation API",
 		Version = "v1"
 	});
+
+	string descriptions = $"{Assembly.GetExecutingAssembly().GetName().Name}.xml";
```

Comment:
Just for my information what does this do?

Reply:
Generates the XML filename for the executing assembly (generated as part of the XML Doc Comment export) MyApp.dll => MyApp.xml